src/map.config.json

```diff
@@ -195,6 +195,55 @@
           }
         }
       },
+      "vectorProperties": {
+        "primitiveOptions": {
+          "type": "cylinder",
+          "geometryOptions": {
+            "length": 1.0,
+            "topRadius": 5.0,
+            "bottomRadius": 5.0,
+            "outline": false,
+            "material": "Cesium.Color.GREEN"
+          },
+          "offset": [0, 0, 0.1]
+        },
+        "modelAutoScale": true
+      },
+      "activeOnStartup": false,
+      "zIndex": 5
+    },
+    {
+      "name": "_trambusStopsOutline",
+      "url": "https://public.sig.rennesmetropole.fr/geoserver/ows?service=WFS&request=getFeature&typename=trp_coll:trambus_arrets&outputFormat=application/json&srsName=EPSG:4326",
```

Comment:
I think there is another way like I did for the arrow.: create an empty layer, add a point for each "shown" POI, and delete them when we are finished.

Honestly, I am not sure which one is the best. I take this approach because:

1. Simpler and more readable in implementation -> easier to maintain (only show or hide features)
2. The features are exactly the same as the station, it's not like in the arrow where the original feature is a point, and I need to create a 3D line
3. I am hoping to have a better performance since we only request it once, compared to creating and deleting features several times on the other approach.

Also, as VCS team mentioned it, currently it's the only way to have a disk rendering like this (i.e. with two-layer)